tests/integration/setup-rendering-context-test.js

```diff
@@ -119,18 +119,10 @@ module('setupRenderingContext "real world"', function(hooks) {
     test('can click on a sibling element of this.element (within the rootElement)', async function(assert) {
       let rootElement = document.getElementById('ember-testing');
 
-      assert.notEqual(
-        rootElement,
-        this.element,
-        'precond - confirm that the rootElement is different from this.element'
-      );
```

Comment:
should we keep this assertion and run it conditionally instead?

Reply:
ya, I'll make it two tests